index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+      <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Poetry</title>
+  </head>
+   <body style="background-color:lightblue;">
+  <body>
+    
+    <p>
+      <h1> 
+  <strong>ON PAPER</strong>
+    </h1> 
+    </p>
+    
+<p>
+ <h2>
+   By Jacqueline Woodson
+  </h2> 
+    </p>
+
+ <p>
+   The first time I write my full name
+    </p>
+<p>
+  Jacqueline Amanda Woodson
+    </p>
+ <p>
+   without anybody's help
+    </p>
+ <p>
+   on a clean white page in my composition notebook,
+    </p>
+ <p>
+   I know  
+     </p>
+<p>
+   if I wanted to
+</p>
+<p>
+  I could write anything.
+    </p>
+<p>
+  Letters becoming words, words gathering meaning,
+    </p>
+<p>
+  becoming thoughts outside my head
+    </p>
+<p> <!--  🧠 🗣 -->
+    </p>  
+<p>
+  becoming sentences
+    </p>
+<p>
+  <p class="written-by">
+  <strong>Written by</strong>
+    </p>
+
+<p>
+  <p class="jac">
+  Jacqueline Amanda Woodson
+</p>  
```

Comment:
This HTML can be simplified; you usually don't want to nest `p` elements inside of each other like this. Try instead:

```suggestion
<p class="written-by">
  <strong>Written by</strong>
</p>

<p class="jac">
  Jacqueline Amanda Woodson
</p>  
```